### PR TITLE
Jupyter Fall 2020 Blog Post

### PR DESCRIPTION
**Pull request by sandertyu**
This should be fine to merge, feel free to let me know if the header info should be changed or you see any spelling errors and whatnot.

### CODE REVIEW

**Comment by moorepants** [comment]
Great! @mandeepika, @TimStewartJ, and @pmackle, review read this and leave any review comments.

File: content/jupyter-fall-2020.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+the webpage. This plugin is completely operational and deployed on libretexts.org.
+
+Cluster Migration
+-----------------
+During the winter break, the Jupyter team migrated the previous Flock cluster 
+to the new Galaxy cluster. This involved various changes in networking,
+hardware, and Jupyterhub.
+
+The first step was to make sure that all of the physical computers were properly
+wired to eachother. At the end of it all, we were able to reconsolidate our Flock
```
Comment (TimStewartJ) [inline content/jupyter-fall-2020.rst:51]: *each other

File: content/jupyter-fall-2020.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+and protogalaxy clusters to where we now have 17 nodes online. These new Galaxy nodes are running
+high availability programs like metallb and HAproxy in order to preserve the availability of
```
Comment (TimStewartJ) [inline content/jupyter-fall-2020.rst:53]: "in order" seems unnecessary

**Review by TimStewartJ — COMMENTED** [review]
looks good!

File: content/jupyter-fall-2020.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+hardware, and Jupyterhub.
+
+The first step was to make sure that all of the physical computers were properly
+wired to eachother. At the end of it all, we were able to reconsolidate our Flock
+and protogalaxy clusters to where we now have 17 nodes online. These new Galaxy nodes are running
+high availability programs like metallb and HAproxy in order to preserve the availability of 
+our Libretexts services and Kubernetes control-planes which is a great improvement over the previous
+Flock cluster.
+
+From a management perspective, the entire cluster was setup using Ubuntu cloudinit to boot the nodes
```
Comment (pmackle) [inline content/jupyter-fall-2020.rst:57]: Change "From a management perspective, the entire cluster..." to something like "We improved cluster management by setting up Ubuntu cloudinit..." to remove passive voice

**Comment by moorepants** [comment]
Thanks everyone! Lots accomplished!

**Pull request merged**